### DISCUSSION TITLE
Two or more post processing directories

### DISCUSSION THREAD

**Discussion by johnvick** [pymedusa/Medusa#10411] [Ideas & Feature requests]
I have /completed for downloads synced from a seedbox and /downloads for manual downloads.  Can I request post processor look at two (or more) directories? I can copy manual downloads to  /completed but this then syncs them back to the seedbox.  I currently manually change post processing directory as required. Thanks for considering.

**Comment by p0psicles** [comment]
Have you considered using the download handler. Then it doest matter where the files are located.

Then you could use the periodic postprocessing for the manual downloaded files?

**Comment by johnvick** [comment]
Thanks - I wasn't aware of that feature ( I assume you mean Automated Download Handling in post-processing section?) but having had a look I am not sure how it would work for me in the above case?

**Reply by p0psicles** [reply]
Oh yeah your disconnected from your torrent client. Yeah, automated download handling won't work in that case.

So for your manual downloads. That torrent client is located on the same machine as Medusa?

**Reply by p0psicles** [reply]
Have you considered using nzbToMedia for the manual downloads?

**Reply by johnvick** [reply]
The torrent client (Deluge) is on the seedbox and the manual downloads are done on a Windows desktop, often from Reddit forums.
